data/mods/Aftershock/items/armor/exosuit/exosuit_frame.json

```diff
@@ -342,7 +342,10 @@
     "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ACTIVE", "values": [ { "value": "STRENGTH", "add": 10 } ] } ] },
     "power_draw": "6173 mJ",
     "revert_to": "modular_exosuit",
-    "use_action": [ { "type": "transform", "menu_text": "Turn off", "msg": "The %s disengages.", "target": "modular_exosuit" } ],
+    "use_action": [
+      { "type": "transform", "menu_text": "Turn off", "msg": "The %s disengages.", "target": "modular_exosuit" },
+      "MANAGE_EXOSUIT"
+    ],
```

Comment:
Adding this back here makes the menu trigger every turn the suit's active.

Reply:
Shoot, didn't notice that, just turned it on and off again. I'll revert this and leave UI changes for another PR.